docs/getting_started.md

```diff
@@ -355,5 +355,8 @@ Please note: We strongly discourage connecting FLUFFI to an untrusted network (s
 
 ## A Note on Operational Security
 
-FLUFFI was not meant to be used in hostile environments, which is why FLUFFI's internal protocol is not hardened against attacks. It is therefore recommended to operate FLUFFI in an isolated environment, to which you can restrict access, and only allow the master to be connected to the internet via NAT.
-Furthermore, it is strongly recommended to change the Polemarch user credentials, which are initially set to be `admin:admin`.
```

Comment:
I understand not having `admin:admin` written directly in the docs, but why remove this line entirely instead of just dropping the part after the comma? Are there any issues that crop up if the polemarch credentials are changed?

Reply:
My thought process when removing this line was something like this:
1. Is this really the only recommendation about security configuration we give here? These are certainly not the only default credentials that we have in the project, and obviously all the others need to be changed as well (think about [these](https://github.com/siemens/fluffi/blob/master/srv/fluffi/data/polenext/projects/1/hosts) and the correspondig documentation [here ](https://github.com/siemens/fluffi/blame/master/docs/getting_started.md#L188))
2.  So why do we need to address this one instance here explicitly?
3. Then I'd better remove this line, because otherwise one might think this is the only thing to take care of when installing FLUFFI.